include/SSVOpenHexagon/Core/MenuGame.hpp

```diff
@@ -103,6 +103,15 @@ class MenuGame
         txtLAuth{"", imagine, 20}, txtLMus{"", imagine, 20},
         txtFriends{"", imagine, 21}, txtPacks{"", imagine, 14};
 
+    // dialog box
+    void drawDialogBox();
+    void clearDialogBox();
+    Utils::FastVertexVector<sf::PrimitiveType::Quads> dialogFrame;
+    std::vector<std::string> dialogText;
+    sf::Text txtDialog{"", imagine, 0};
+    float dialogHeight{0.f}, dialogWidth{0.f}, frameOffset{0.f}, lineHeight{0.f};
+    int noActions{0};
+
```

Comment:
How about putting all of this stuff into its own class? Seems quite self-contained, and it could be used for other dialog boxes